packages/components/src/text/types.ts

```diff
@@ -59,7 +60,7 @@ export interface Props extends TruncateProps {
 	/**
 	 * Array of search words. String search terms are automatically cast to RegExps unless `highlightEscape` is true.
 	 */
-	highlightSanitize?: import('highlight-words-core').FindAllArgs[ 'sanitize' ];
+	highlightSanitize?: FindAllArgs[ 'sanitize' ];
```

Comment:
Not sure if it should be part of this PR or done as a follow-up, but just wanted to note that `highlight-words-core` is also used by [packages/components/src/text/utils.js](https://github.com/WordPress/gutenberg/blob/33d84b036592a5bf31af05b7710f3b2b14163dc4/packages/components/src/text/utils.js), which at this point could do with a TS refactor